src/cascadia/TerminalApp/lib/TerminalAppLib.vcxproj

```diff
@@ -330,8 +330,4 @@
   <Target Name="_TerminalAppGenerateUserSettingsH" Inputs="..\userDefaults.json" Outputs="Generated Files\userDefaults.h" BeforeTargets="BeforeClCompile">
     <Exec Command="powershell.exe -noprofile –ExecutionPolicy Unrestricted $(OpenConsoleDir)\tools\GenerateHeaderForJson.ps1 -JsonFile ..\userDefaults.json -OutPath '&quot;Generated Files\userDefaults.h&quot;' -VariableName UserSettingsJson" />
   </Target>
-  <!-- Get the hash of the current git commit this build is on.-->

```

Comment:
RIP in peace, I loved having this in the repo. I figured we'd use it in the future for stamping into the dev build so you could know "I'm running the version based off my commit `f00ba7` but I get it

Reply:
I could not-revert this bit if you want. Right now it's based on the age of the NOTICES file, which doesn't change w/ every commit...

It might be better to tackle this as part of our all-up versioning story later?

Reply:
Eh, I don't care enough. It's in the history of the repo, not too hard to resurrect.